docs/rules-engine/how-to-use/dedicated-component-ruleset.md

```diff
@@ -18,60 +63,41 @@ First, you need to have a ruleset defined in your json (created via the CMS inte
+        // and we will add the support
+        "or": [ 
           {
             "library": "@scope/components",
-            "name": "TestComponent"
+            "name": "MyComponent"
           },
           {
             "library": "@scope/components",
-            "name": "TestComponent2"
+            "name": "MyComponent2"
           }
         ]
       },
       "rules": [
-        ...
+        // ...
       ]
     }
   ]
 }
 ```
-> Note: In v10 and previously, we used `linkedComponent` property to activate a ruleset on demand. This becomes deprecated and will be removed in v12. Use `linkedComponents` instead;
-
-Now that you have the id, you will need to subscribe to the ruleset in your component :
-
-```typescript
-import {LinkableToRuleset, RulesEngineRunnerService} from '@o3r/rules-engine';
-import {computeItemIdentifier, type ExposedComponent} from '@o3r/core';
-
-export class MyComponent implements LinkableToRuleset, ExposedComponent {
-
-  private readonly componentIdentifier = computeItemIdentifier('@scope/components', 'TestComponent');
-
-  constructor(@Optional() private readonly service: RulesEngineRunnerService) {
-  }
-
-  public ngOnInit() {
-    this.service?.enableRuleSetFor(componentIdentifier);
-  }
-
-  public ngOnDestroy() {
-    this.service?.disableRuleSetFor(componentIdentifier);
-  }
-}
-```
-
-Important : Please note that you need your component either to be a Page, Block or an ExposedComponent for it to be extracted in the metadata file and sent to the CMS.
+> [!INFO]
+> In v10 and the previous version, the Otter framework exposed the `linkedComponent` property to activate a ruleset on demand. 
+> This is now deprecated and will be removed in v12. Use `linkedComponents` instead.
 
-## How does it work
+## How it works
 
-The RulesEngineRunnerService is the one that will handle the default subscriptions.
-At the initialization of the application, the ruleset store will provide a selector to get all Active ruleset.
-Then this selector will be used to subscribe to those rulesets in the service as an initial behavior.
-Our onDemand ruleset will not be part of this initial subscription, that's why as long as no one subscribes to it, it will not be evaluated.
-As soon as our component is instantiated on the page (ngOnInit called), it will explicitly subscribe to it, resulting in its evaluation (and refresh if any input facts changes).
-Once the subscription is removed (ngOnDestroy) called, and no other component is linked to it, the ruleset will go back to its previous 'disabled' mode.
+`RulesEngineRunnerService` handles the subscriptions for all the rulesets.
+At the initialization of the application, the service listens to a selector to get all the active rulesets that are not linked
+to any components -- note that they may have a validity date range.
```

Comment:
could we add a small section with some words about this feature. If there is one, you can ignore this comment :)